tests/builders/test_builders.py

```diff
@@ -138,3 +138,15 @@ def test_file_count():
         configuration=config,
     )
     cmd.build()
+
+def test_file_regex():
+    """This test buildspec using status check with 'file_regex'"""
+    cmd = BuildTest(
+        buildspecs=[
+            os.path.join(here, "file_regex.yml")
```

Comment:
line 146: These is no file_regex.yml file in tutorials/test_status to link to, please make sure the name of yml file is correct.